testsuites/CBLTester/CBL_Functional_tests/TestSetup_FunctionalTests/test_replicator_encryption_hook.py

```diff
@@ -0,0 +1,107 @@
+import pytest
+import time
+
+from keywords.MobileRestClient import MobileRestClient
+# from keywords.ClusterKeywords import ClusterKeywords
+# from keywords import couchbaseserver
```

Comment:
if those commented-out import statements are not needed, it's better to clean them out